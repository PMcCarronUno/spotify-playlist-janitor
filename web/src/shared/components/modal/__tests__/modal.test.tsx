import { render } from "@testing-library/react";

import { Modal } from "..";

describe("<Modal />", () => {
  let baseElement: HTMLElement;
  let props: any;
  
  beforeEach(() => {
    props = {
      label: "Modal",
      isOpen: true,
      onClose: jest.fn(),
    };

    ({ baseElement } = render(
      <Modal {...props}><div>Modal Content</div></Modal>
    ));
  });

  it("should render Modal component", () => {
    expect(baseElement.firstChild).toMatchSnapshot();
  });
});
