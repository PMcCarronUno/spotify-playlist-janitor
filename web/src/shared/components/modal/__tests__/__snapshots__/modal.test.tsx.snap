// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> should render Modal component 1`] = `
<div
  data-react-modal-body-trap=""
  style="position: absolute; opacity: 0;"
  tabindex="0"
/>
`;
